balancer/weightedroundrobin/balancer.go

```diff
@@ -608,7 +608,7 @@ func (w *weightedSubConn) weight(now time.Time, weightExpirationPeriod, blackout
 
 	// The SubConn has not received a load report (i.e. just turned READY with
 	// no load report).
-	if w.lastUpdated == (time.Time{}) {
+	if time.Time.Equal(w.lastUpdated, time.Time{}) {
```

Comment:
What is `time.Time`?

Why not one of the two options?
- `w.lastUpdate.Equal(time.Time{})`
- `w.lastUpdate.IsZero()`

Reply:
gotcha, updated all the occurences in the file.